homeassistant/components/pjlink/media_player.py

```diff
@@ -181,3 +305,74 @@ def select_source(self, source):
         source = self._source_name_mapping[source]
         with self.projector() as projector:
             projector.set_input(*source)
+
+    @property
+    def extra_state_attributes(self):
+        """Add the extra projector specific attributes."""
+        state_attr = {}
+
+        for attr in ATTR_TO_PROPERTY:
+            if (value := getattr(self, attr)) is not None:
+                state_attr[attr] = value
```

Comment:
Many of these (like lamp status, fan errors),  should not be exposed as attributes; instead, they should become their own (binary) sensor entities.